### PR TITLE
Show all users items

### PR DESCRIPTION
**Pull request by MariusKurbakovas**
Review (fast if you want to see all items list before demo)

### CODE REVIEW

File: GREEDY/Controllers/ItemsController.cs [hunk 1]
```diff
@@ -33,6 +33,29 @@ public HttpResponseMessage Get(int id)
     }
 
     [EnableCors(origins: "*", headers: "*", methods: "*")]
+    public class GetAllUserItemsController : ApiController
+    {
```
Comment (Upsiukas) [inline GREEDY/Controllers/ItemsController.cs:37]: The name of the class sounds weird.

File: GREEDY_WEB/ClientApp/components/ChooseFiltersForItems.tsx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+import { ModalContainer, ModalDialog } from 'react-modal-dialog';
+import Constants from './Constants';
+import { EditItem } from './EditItem';
+
+interface Props {
+    refilter: any
+}
+
+interface State {
+    showChoose: boolean
```
Comment (Upsiukas) [inline GREEDY_WEB/ClientApp/components/ChooseFiltersForItems.tsx]: Maybe this should be showChoice

**Review by Upsiukas — APPROVED** [review]
The C# part looks good, I dont understand react that much but the code looks readable, so thats good

File: GREEDY/Controllers/ItemsController.cs (hunk 1 above)
Comment (MariusKurbakovas) [reply GREEDY/Controllers/ItemsController.cs:37]: Imagine that it is GetAllUserItems. Does that sound good to you? (The controller part is neccesary to make HTTP requests reach it). So I think everything is good here

**Pull request merged**